code_challenges.rb

```diff
@@ -1,19 +1,67 @@
 # ASSESSMENT 4: Ruby Coding Practical Questions
 
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest.
-
-
-
-
-
 fullArr1 = [4, 9, 0, '7', 8, true, 'hey', 7, 199, -9, false, 'hola']
 # Expected output: [-9, 7, 9, 199]
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
+def odd_num(arr)
+    sorted_arr = []
+    arr.filter do |value|
+        if value.is_a? Integer
+        sorted_arr << value 
+        end
+    end
+        arr = []
+        sorted_arr.filter do |value|
+            if value % 2 == 1
+            arr << value 
+            end
+        end
+    arr.sort
+end
+print odd_num(fullArr1)
+puts ""
+print odd_num(fullArr2)
+puts ""
```

Comment:
yay! good job! you can also just put `p odd_num(fullArr1)` instead of `print` to avoid having to use `puts ""` to start a new line ;)